specification/azurestackhci/resource-manager/Microsoft.AzureStackHCI/stable/2023-09-01/marketplaceGalleryImages.json

```diff
@@ -0,0 +1,595 @@
+        },
+        "parameters": [
+          {
+            "$ref": "../../../../../common-types/resource-management/v3/types.json#/parameters/SubscriptionIdParameter"
+          },
+          {
+            "$ref": "../../../../../common-types/resource-management/v3/types.json#/parameters/ResourceGroupNameParameter"
+          },
+          {
+            "$ref": "#/parameters/MarketplaceGalleryImageNameParameter"
+          },
+          {
+            "$ref": "../../../../../common-types/resource-management/v3/types.json#/parameters/ApiVersionParameter"
+          }
+        ],
+        "responses": {
+          "default": {
+            "description": "Error response describing why the operation failed.",
+            "schema": {
+              "$ref": "../../../../../common-types/resource-management/v3/types.json#/definitions/ErrorResponse"
+            }
+          },
+          "200": {
+            "description": "Deleted the marketplace gallery image resource."
+          },
+          "202": {
+            "description": "Accepted",
+            "headers": {
+              "Location": {
+                "type": "string"
+              }
+            }
+          },
+          "204": {
+            "description": "No content"
+          }
+        }
+      },
+      "patch": {
+        "operationId": "MarketplaceGalleryImages_Update",
+        "description": "The operation to update a marketplace gallery image.",
+        "x-ms-examples": {
+          "UpdateMarketplaceGalleryImage": {
+            "$ref": "./examples/UpdateMarketplaceGalleryImage.json"
+          }
+        },
+        "x-ms-long-running-operation": true,
+        "x-ms-long-running-operation-options": {
+          "final-state-via": "azure-async-operation"
+        },
+        "parameters": [
+          {
+            "$ref": "../../../../../common-types/resource-management/v3/types.json#/parameters/SubscriptionIdParameter"
+          },
+          {
+            "$ref": "../../../../../common-types/resource-management/v3/types.json#/parameters/ResourceGroupNameParameter"
+          },
+          {
+            "$ref": "#/parameters/MarketplaceGalleryImageNameParameter"
+          },
+          {
+            "$ref": "../../../../../common-types/resource-management/v3/types.json#/parameters/ApiVersionParameter"
+          },
+          {
+            "name": "marketplaceGalleryImages",
+            "in": "body",
+            "required": true,
+            "schema": {
+              "$ref": "#/definitions/MarketplaceGalleryImagesUpdateRequest"
+            }
+          }
+        ],
+        "responses": {
+          "default": {
+            "description": "Error response describing why the operation failed.",
+            "schema": {
+              "$ref": "../../../../../common-types/resource-management/v3/types.json#/definitions/ErrorResponse"
+            }
+          },
+          "200": {
+            "description": "OK",
```

Comment:
>OK [](http://example.com/codeflow?start=28&length=2)

Should perhaps say something a little more context specific like, 'The update succeeded?

Reply:
We modeled against compute for this: https://github.com/Azure/azure-rest-api-specs-pr/blob/de06c42bd985fb66b6a49c907aaf5baee693bc7b/specification/compute/resource-manager/Microsoft.Compute/ComputeRP/stable/2023-07-01/image.json#L168

Won't fix.